modules/c++/re/tests/benchmark_regex.cpp

```diff
@@ -0,0 +1,96 @@
+
+#include <string>
+#include <fstream>
+
+#include <import/re.h>
+#include <sys/StopWatch.h>
+#include <sys/LocalDateTime.h>
+
+// First just benchmark the Regex creation time (including compile)
+double BM_RegexCreation(uint64_t numIterations)
+{
+    sys::RealTimeStopWatch sw;
+
+    sw.start();
+    for (uint64_t ii = 0; ii < numIterations; ++ii)
+    {
+        re::Regex regex("beam(Id|String)");
```

Comment:
Hmm, I guess we should take this string in on the command line too so that we can tailor it to what we're parsing